charts/latest/csi-driver-nfs/templates/csi-nfs-node.yaml

```diff
@@ -79,12 +83,11 @@ spec:
             - name: registration-dir
               mountPath: /registration
           resources: {{- toYaml .Values.node.resources.nodeDriverRegistrar | nindent 12 }}
-        - name: nfs
+          {{- with .Values.node.containerSecurityContext.nodeDriverRegistrar }}
           securityContext:
```

Comment:
currently only `nfs` container needs `securityContext`, I don't think it's necessary to set `securityContext` for every sidecar container

Reply:
We want to set readonlyrootfilesystem. You have set it on every container.

Reply:
then just set `readOnlyRootFilesystem` for other containers by default, and only need to set `.Values.node.securityContext` and `.Values.controller.securityContext` for nfs container.

Reply:
I changed the behavior